### PR TITLE
Add Hive/Hive2 Mapper

### PR DESCRIPTION
**Pull request by mik-laj**
Make sure you have checked _all_ steps below and that above description is correct.

### JIRA
- [ ] My PR addresses the following [Issue](https://github.com/GoogleCloudPlatform/oozie-to-airflow/issues/28)
issue and references it in commit message.

### Tests
- [ ] My PR has unit tests testing the functionality (or I explained why it needs no tests)

### Commits
- [ ] My commits reference the issue in the message using: Fixes #XXX /Closes #XXX or similar.

### Documentation
- [ ] My PR adds documentation that describes how to use it (or I explained why it needs no docs)

### CODE REVIEW

File: o2a/utils/param_extractor.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+from o2a.utils import xml_utils, el_utils
+
+TAG_PARAM = "param"
+
+
+def extract_param_values_from_action_node(action_node, props):
+    param_nodes = xml_utils.find_nodes_by_tag(action_node, TAG_PARAM)
+
+    if not param_nodes:
+        return None
```
[the remark this answers is not in the document]
Comment (mik-laj) [reply o2a/utils/param_extractor.py]: I updated the PR. Thanks.

File: o2a/utils/param_extractor.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Extract params from oozie's action node"""
+from o2a.utils import xml_utils, el_utils
+
+TAG_PARAM = "param"
+
+
+def extract_param_values_from_action_node(action_node, props):
```
[the remark this answers is not in the document]
Comment (mik-laj) [reply o2a/utils/param_extractor.py]: I updated the PR. Thanks.

File: tests/utils/test_param_extractor.py [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+
+
+class ParamExtractorModuleTestCase(unittest.TestCase):
+    def test_extract_param_values_from_action_node_should_return_none_when_not_found(self):
+        props = PropertySet(
+            config={}, job_properties={"userName": "TEST_USERNAME", "examplesRoot": "TEST_EXAMPLE_ROOT"}
+        )
+        node = ET.fromstring("<dummy></dummy>")
+        result = extract_param_values_from_action_node(node, props=props)
+        self.assertEqual(None, result)
```
[the remark this answers is not in the document]
Comment (mik-laj) [reply tests/utils/test_param_extractor.py]: I updated the PR. Thanks.

**Comment by potiuk** [comment]
Feel free to merge it @mik-laj

**Pull request merged**